### PR TITLE
BETA: Register verification pub.is-a.dev

### PR DESCRIPTION
**Pull request by D4rk-cipher**
s

### CODE REVIEW

File: domains/verification pub.json
Comment (Lukas-Batema) [inline domains/verification pub.json:1]: Please rename your file here to use something that can be used in a url.

**Review by luni-moon — CHANGES_REQUESTED** [review]

File: domains/west.exch039.server.json
Comment (WilliamDavidHarrison) [inline domains/west.exch039.server.json:1]: You will need to register `server.is-a.dev` and `exch039.server.is-a.dev` first. Also, what is this domain being used for?

File: domains/west.verification.server.json
Comment (WilliamDavidHarrison) [inline domains/west.verification.server.json:1]: You will need to register `server.is-a.dev` and `verification.server.is-a.dev` first. Also, what is this domain being used for?

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please also see @Lukas-Batema's review comment.

**Pull request closed without merging**